### PR TITLE
Upgrade API Client with the lates version of Fireblocks OpenAPI specification

### PR DESCRIPTION
**Pull request by joaofbvieira**
This PR will generate a major release of `Trakx.Fireblocks.ApiClient`, since the OpenAPI specification changed a lot comparing with our current version.
The original OpenAPI was kept too for future reference, since it was containing some bugs and conflicting model names (Task for example).

### CODE REVIEW

File: tests/Trakx.Fireblocks.ApiClient.Tests/Integration/ExchangeAccountsClientTests.cs [hunk 1]
```diff
@@ -16,7 +16,7 @@ public ExchangeAccountsClientTests(FireblocksApiFixture apiFixture, ITestOutputH
     [Fact]
     public async Task Exchange_accountsAllAsync_should_return_all_accounts()
     {
-        var response = await _exchangeAccountsClient.Exchange_accountsAllAsync();
-        response.Content.Should().BeEmpty();
+        var response = await _exchangeAccountsClient.GetPagedExchangeAccountsAsync(5);
```
Comment (joaofbvieira) [inline tests/Trakx.Fireblocks.ApiClient.Tests/Integration/ExchangeAccountsClientTests.cs:19]: The old method is obsolete, so we must use the paged one.

File: tests/Trakx.Fireblocks.ApiClient.Tests/Integration/SupportedAssetsClientTests.cs [hunk 2]
```diff
@@ -6,22 +6,21 @@ namespace Trakx.Fireblocks.ApiClient.Tests.Integration;
 public class SupportedAssetsClientTests : FireblocksClientTestsBase
 {
 
-    private readonly ISupported_assetsClient _supportedAssetsClient;
+    private readonly IBlockchains_and_assetsClient _supportedAssetsClient;
```
Comment (joaofbvieira) [inline tests/Trakx.Fireblocks.ApiClient.Tests/Integration/SupportedAssetsClientTests.cs:9]: This client was completely renamed.

File: tests/Trakx.Fireblocks.ApiClient.Tests/Integration/TransactionsClientTests.cs [hunk 3]
```diff
@@ -25,7 +26,7 @@ public async Task CreateTransactionAsync_should_create_and_cancel_a_new_transact
     {
         var trans = new TransactionRequest
         {
-            Amount = 0.001,
+            Amount = "0.001",
```
Comment (joaofbvieira) [inline tests/Trakx.Fireblocks.ApiClient.Tests/Integration/TransactionsClientTests.cs:29]: One of the biggest breaking changes is that the old "Amount" became a string.
In the response, the "Amount" is obsolete, where we must use the "AmountInfo" now, which all value properties are string too.

File: tests/Trakx.Fireblocks.ApiClient.Tests/Integration/VaultClientTests.cs [hunk 4]
```diff
@@ -16,7 +16,7 @@ public VaultClientTests(FireblocksApiFixture apiFixture, ITestOutputHelper outpu
     [Fact]
     public async Task GetVaultAccountsAsync_should_return_all_vault_accounts()
     {
-        var response = await _vaultClient.Accounts_pagedAsync();
+        var response = await _vaultClient.GetPagedVaultAccountsAsync();
```
Comment (joaofbvieira) [inline tests/Trakx.Fireblocks.ApiClient.Tests/Integration/VaultClientTests.cs:19]: The old method is obsolete, so we must use the paged one.

File: tests/Trakx.Fireblocks.ApiClient.Tests/Integration/WorkflowTests.cs [hunk 5]
```diff
@@ -35,7 +36,7 @@ public async Task Sending_across_network_connections_should_work()
         var transactionRequest = new TransactionRequest
         {
             FailOnLowFee = true,
-            Amount = 0.01d,
+            Amount = "0.01",
```
Comment (joaofbvieira) [inline tests/Trakx.Fireblocks.ApiClient.Tests/Integration/WorkflowTests.cs:39]: One of the biggest breaking changes is that the old "Amount" became a string.
In the response, the "Amount" is obsolete, where we must use the "AmountInfo" now, which all value properties are string too.

File: tests/Trakx.Fireblocks.ApiClient.Tests/Unit/FireblocksApiClientsFactoryTests.cs [hunk 6]
```diff
@@ -11,8 +11,8 @@ public class FireblocksApiClientsFactoryTests
     [ClassData(typeof(FireblocksApiClientsFactoryTestsData))]
     public void CreateApiClient_should_create_new_client_with_given_credentials(
         Func<FireblocksApiCredentialsConfiguration, IFireblocksApiClientBase> func,
-        Type expectedInterfaceType,
-        Type expectedImplementationType)
+        System.Type expectedInterfaceType,
```
Comment (joaofbvieira) [inline tests/Trakx.Fireblocks.ApiClient.Tests/Unit/FireblocksApiClientsFactoryTests.cs:14]: There is a model named `Type`, generated automatically by NSwag, so we might face some conflicts, being a possible breaking change too.

File: tests/Trakx.Fireblocks.ApiClient.Tests/Unit/Serialisation/ExchangeAccountTypeSerialisationTests.cs [hunk 7]
```diff
@@ -8,6 +8,6 @@ public void Should_Serialise_exchange_response_back_and_forth()
     {
         var responseContent = File.ReadAllText("Unit/Serialisation/sampleGetAllExchangesResponse.json");
         var exchanges = Newtonsoft.Json.JsonConvert.DeserializeObject<ExchangeAccount[]>(responseContent);
-        exchanges!.Length.Should().Be(9);
+        exchanges!.Length.Should().Be(7);
```
Comment (joaofbvieira) [inline tests/Trakx.Fireblocks.ApiClient.Tests/Unit/Serialisation/ExchangeAccountTypeSerialisationTests.cs:11]: 2 exchanges in the json file were using invalid enum values for this version (KUCOIN and GATEIO; COINBASEEXCHANGE was renamed to COINBASEPRO).

File: tests/Trakx.Fireblocks.ApiClient.Tests/Unit/FireblocksApiClientsFactoryTests.cs (hunk 6 above)
Comment (monsieurleberre) [reply tests/Trakx.Fireblocks.ApiClient.Tests/Unit/FireblocksApiClientsFactoryTests.cs:14]: nooooo 😭

**Review by monsieurleberre — APPROVED** [review]
Yes 💪

**Pull request merged**